src/sentry/static/sentry/app/views/organizationIncidents/details/header.jsx

```diff
@@ -44,7 +46,7 @@ export default class DetailsHeader extends React.Component {
           label={incident && <Status incident={incident} />}
           menuWidth="160px"
           alignRight
-          buttonProps={!incident ? {disabled: true} : null}
+          buttonProps={!incident ? {disabled: true, size: 'small'} : {size: 'small'}}
```

Comment:
I would do something like this

```suggestion
          buttonProps={{size: 'small', disabled: !incident}}
```